src/locales/it/translation.json

```diff
@@ -328,129 +328,129 @@
+            "weaknessTokens": "Pesca 3 Segnalini Luogo (1 per ogni terreno). Se in quel luogo si trova un Witcher, pesca di nuovo. Posiziona un Segnalino Debolezza Mostro casuale e a faccia in giù corrispondente a quel tipo di terreno in ognuno di quei luoghi. Rimescola i Segnalini Luogo nelle loro pile."
         },
         "adventurePack": {
-            "tokens": "Place Gold Tokens, Green Quest Tokens, Poker Dice, and Closed Tavern token or mini near the game board.",
-            "lostMount": "The last player draws two Location Tokens (of any 1 type of terrain), chooses one to put the Lost Mount miniature at, and shuffles the tokens back in."
+            "tokens": "Posiziona i Segnalini Oro, Missione Verde, i dadi da poker e i Segnalini - o la miniatura - Taverna Chiusa vicino al tabellone di gioco.",
+            "lostMount": "L'ultimo giocatore pesca due Segnalini Luogo (di un qualsiasi tipo di terreno), ne sceglie uno su cui posizionare la miniatura del Destriero Perduto e rimescola nuovamente i segnalini."
         }
     },
     "inventoryChecker": {
-        "title": "Inventory Checker",
-        "cardSubtitle": "Cards",
-        "noteTitle": "Notes",
+        "title": "Controllo Componenti",
+        "cardSubtitle": "Carte",
+        "noteTitle": "Note",
         "noteText": [
-            "This is currently a first pass to get this inventory checker into the hands of the second wave of shipments as fast as possible, and as such, will be minimal and (probably) lacking translation.",
-            "I aim to add different sections for the tokens and miniatures shortly after this goes live.",
-            "Currently, the check boxes are not persistent. If you refresh or navigate away from this page, you will lose your progress. Apologies for the inconvenience while I sort this out."
+            "Questa è una prima bozza per far arrivare il Controllo Componenti nelle mani della seconda ondata di spedizioni il più velocemente possibile e, come tale, sarà minimale e (probabilmente) privo di traduzione.",
+            "Il mio obiettivo è aggiungere diverse sezioni per i segnalini e le miniature subito dopo la pubblicazione.",
+            "Al momento le spunte di controllo non sono permanenti. Se aggiorni o esci da questa pagina perderai i tuoi progressi. Mi scuso per l'inconveniente mentre risolvo la questione."
         ],
-        "legendHeader": "Legend",
+        "legendHeader": "Legenda",
         "legendBody": {
-            "header": "If two columns: read left to right, top to bottom. This will ensure the running totals of cards spread across multiple packs (e.g. \"Monster cards: 11 (11/29)\") are read properly.",
+            "header": "Se su due colonne: leggi da sinistra a destra, dall'alto in basso. Ciò garantirà che i totali parziali delle carte distribuite su più mazzi (ad esempio \"Carte Mostro: 11 (29/11)\") vengano letti correttamente.",
```

Comment:
Does italian count items out of a total number of items in a flipped order, e.g. "29 total, 11 so far"?

The value here, "11/29", is equal to "11 out of the total 29 items so far". I'm assuming this was an accidental recognition of this being a date, November 29, and flipping it to be accurate for Italian, do I have that correct?